src/graphql/schemas/user-schema.ts

```diff
@@ -38,6 +38,7 @@ export const userTypeDefs = gql`
 
   type Query {
     user(id: ID!): User
+    users(limit: Int): [User]
```

Comment:
Poderia set `[User!]!` para forçar o retorno de um array de usuários em vez de deixar nullable